### ISSUE TITLE
[MAS1.3.1]: List is not associated to group header.

### ISSUE THREAD

**Issue by bcoard** [microsoft/fluentui#13405]
User Impact: 
People with Visual Impairments will be unable to know the group/header name while navigated to the items under it.

Build Details: ​
OS Version: 19628.1
AT: Narrator
Edge Version: 84.0.508.0 (Official build) dev (32-bit) ​
​​
Repro Steps: ​
1. Tab to navigate to any of the Group header and Press enter key to expand it.
2. Now use 'Up/Down' arrow keys to navigate to a list item.
3. Tab to navigate out of the list so that focus falls to another control.
4. Once again press 'Shift+Tab' to navigate back to the list and listen to the narrator announcement here. Narrator is announcing as "<name>, <selection state>".

Actual Result: ​
On navigating to the list of under the header group, Narrator is announcing as "<name>, <selection state>". Narrator fails to announce the group name labeled to the list items.
If the user navigates out of the list and navigates back, user won't understand which group they are in.

Expected Result: ​
When navigating back to the list the user should get the contextual information of what header group they are in.

Note: Part of this fix might be to label a group header with the role "group".

**Comment by JustSlone** [comment]
@smhigley it seems like this should be fixed by our recent updates to DetailsList, if so let's close it.
